lisa/tools/wsl.py

```diff
@@ -0,0 +1,332 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import os
+import re
+import time
+from pathlib import PurePath
+from typing import TYPE_CHECKING, Any, Dict, Optional
+
+import charset_normalizer
+from assertpy.assertpy import assert_that
+
+from lisa.base_tools.wget import Wget
+from lisa.executable import Tool
+from lisa.util import LisaException, create_timer, find_groups_in_lines
+from lisa.util.process import ExecutableResult, Process
+
+from . import Echo, Find, Ls, PowerShell, Tar
+
+if TYPE_CHECKING:
+    from lisa.node import Node
+
+
+class Wsl(Tool):
+    """
+    Support commands on WSL host and WSL both.
+    """
+
+    FIXED_WSL_PATH = r"%ProgramFiles%\wsl\wsl.exe"
+    CONFIG_FILE_PATH = r"%USERPROFILE%\.wslconfig"
+
+    ENCODING = "utf-16-le"
+    LKG_RELEASE = "https://github.com/microsoft/WSL/releases/download/2.0.4/Microsoft.WSL_2.0.4.0_x64_ARM64.msixbundle"  # noqa: E501
```

Comment:
should it be passed from runbook?

Reply:
The version is well tested with current logic. The behavior maybe changed on new versions. Finally, it should be removed, once the newer versions are released.